guides/v2.3/graphql/reference/configurable-product.md

```diff
@@ -76,8 +95,8 @@ The following query returns information about configurable product `WH01`, which
           product {
             id
             categories {
```

Comment:
The categories object produces redundant output. Let's save a few electrons by removing that object from the query.

I can't comment on text that isn't near changed text, but also delete store_label, default_label, and use_default_value (lines 88-90).